src/pages/ViewScholarship.jsx

```diff
@@ -47,6 +49,30 @@ export default function ViewScholarship({ history, location, match }) {
       <Helmet>
         <title>{scholarship.data.name}</title>
       </Helmet>
+
+      {location?.state?.status && (
+        <Alert
+          color="primary"
+          variant="filled"
+          action={
+            <>
+              <IconButton
+                size="medium"
+                color="inherit"
+                onClick={() =>
+                  history.replace(`/scholarships/${scholarship.id}`)
+                }>
+                <CloseIcon fontSize="small" />
+              </IconButton>
+            </>
+          }>
+          <AlertTitle sx={{ color: '#fff' }}>Success</AlertTitle>
```

Comment:
Why was the color styling necessary?

Reply:
<img width="1157" alt="Screen Shot 2021-12-20 at 11 55 24 PM" src="https://user-images.githubusercontent.com/40483569/146892399-2c3c0262-cc46-47cd-aeaf-6650ba5df2c9.png">

title was not visible

Reply:
removed title.